src/mono/browser/runtime/cwraps.ts

```diff
@@ -228,6 +229,7 @@ export interface t_Cwraps {
     mono_jiterp_type_get_raw_value_size(type: MonoType): number;
     mono_jiterp_parse_option(name: string): number;
     mono_jiterp_get_options_as_json(): number;
+    mono_jiterp_get_option_as_int(name: string): number;
```

Comment:
this cwrap would be allocating buffer and re-encoding chars on each call. You can pre-allocate because they are constant strings. Or even better use numeric enum instead of name.

Reply:
Absolutely, but under normal circumstances it only runs once. I am open to numeric enum, it's just more complexity.

Reply:
could you please explain what are the not "normal circumstances" ?

Reply:
if you change jiterpreter options at runtime, it will invalidate the mono options data and jiterp will re-fetch all the options. but under normal circumstances that will only happen if the jiterpreter shuts itself off due to failures or oom.

Reply:
`getOptions()` is called quite often. Is `mono_jiterp_get_options_version` really cheap ?

Reply:
1 out of ~390000 startup profile samples is mono_jiterp_get_options_version, so I think it's cheap enough